### ISSUE TITLE
inherited scripts are overwritten

### ISSUE THREAD

**Issue by GCHQDeveloper211** [olofk/fusesoc#646]
It seems that if you have a core file that creates a post_build hook with a script, lets say FOO. If you coincidentally create a post_build hook and script also called FOO, either by inheriting from another core file or creating it in the core file in question, this will create a makefile with several targets all with the same name, and hence only the latter target will be run.
since this is done in the make file, is it worth having fusesoc embed a part of the VLNV or some sort of identifier to ensure that all the hook targets are unique?

**Comment by olofk** [comment]
Yes, some kind of protection is apparently needed. And if I understand correcty the fix needs to bee on the FuseSoC side. Patches welcome!
